src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs

```diff
@@ -42,12 +46,21 @@ public virtual Decision IsSatisfiedBy(RemoteMovie subject, SearchCriteriaBase se
             _logger.Debug("Performing history status check on report");
             _logger.Debug("Checking current status of movie [{0}] in history", subject.Movie.Id);
             var mostRecent = _historyService.MostRecentForMovie(subject.Movie.Id);
+            var customFormats = _formatService.ParseCustomFormat(mostRecent);
```

Comment:
Movie into the `if` statement below so that we only call if mostRecent is not null and is a grabbed event. Prevents calling and calculating for nothing if the statement fails, and prevents null in format calculate service if mostRecent is null.